src/nl/hannahsten/texifyidea/completion/pathcompletion/LatexGraphicsPathProvider.kt

```diff
@@ -0,0 +1,43 @@
+package nl.hannahsten.texifyidea.completion.pathcompletion
+
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.psi.PsiFile
+import nl.hannahsten.texifyidea.psi.LatexNormalText
+import nl.hannahsten.texifyidea.util.childrenOfType
+import nl.hannahsten.texifyidea.util.files.commandsInFileSet
+import nl.hannahsten.texifyidea.util.files.findRootFile
+import java.io.File
+
+/**
+ * @author Lukas Heiligenbrunner
+ */
+class LatexGraphicsPathProvider : LatexPathProviderBase() {
+    override fun selectScanRoots(file: PsiFile): ArrayList<VirtualFile> {
+        val paths = getProjectRoots()
+
+        // Add last included graphicspaths.
+        val graphicsPath = file.commandsInFileSet().filter { it.commandToken.text == "\\graphicspath" }
+        if (graphicsPath.isNotEmpty()) {
+            graphicsPath.last().parameterList.first { firstParam -> firstParam.requiredParam != null }
+                    .childrenOfType(LatexNormalText::class).forEach { path ->
+                        // Check if graphicspath is an absolute or relative path
+                        if (File(path.text).isAbsolute) {
+                            file.findRootFile().containingDirectory.virtualFile.fileSystem.findFileByPath(path.text)?.apply {
```

Comment:
In all places where you added `file.findRootFile().containingDirectory` we need to protect against it's being null, see #1243. 
See the contract of getContainingDirectory(): `@return the containing directory, or {@code null} if the file exists only in memory.`